Dfe.Data.SearchPrototype/Web/Tests/APITests/ApiSearchResults.cs

```diff
@@ -0,0 +1,113 @@
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+
+namespace Dfe.Data.SearchPrototype.WebApi.Tests.APITests;
+
+public class ApiSearchResults : IClassFixture<PageWebApplicationFactory<Program>>
+{
+    private const string SEARCHKEYWORD_ENDPOINT = "/establishments?SearchKeyword=";
+    private readonly HttpClient _client;
+    private readonly ITestOutputHelper _logger;
+    private readonly PageWebApplicationFactory<Program> _factory;
+
+    public ApiSearchResults(PageWebApplicationFactory<Program> factory, ITestOutputHelper logger)
+    {
+        _client = factory.CreateClient(new WebApplicationFactoryClientOptions
+        {
+            AllowAutoRedirect = true
+        });
+        _factory = factory;
+        _logger = logger;
+    }
+
+    [Fact]
+    public async Task GetSwaggerPage()
+    {
+        var response = await _client.GetAsync("/swagger/index.html");
+
+        response.EnsureSuccessStatusCode();
+    }
+
+    [Theory]
+    [InlineData("Academy", 2)]
+    [InlineData("School", 1)]
+    public async Task GET_Search_Returns_ExpectedNumber_Of_Results(string query, int resultsInt)
+    {
+        // all searches have to have the '*' added for now - this needs to go to the top of the list for refactor
+        var response = await _client.GetAsync($"{SEARCHKEYWORD_ENDPOINT}{query}*");
+
+        var responseBody = await response.Content.ReadAsStringAsync();
+        var jsonString = JsonConvert.DeserializeObject<SearchResultsJson>(responseBody)!;
+
+        jsonString.EstablishmentResults!.Establishments.Should().HaveCount(resultsInt);
+    }
+
+    [Fact]
+    public async Task GET_Search_Returns_EstablishmentData()
+    {
+        var response = await _client.GetAsync($"{SEARCHKEYWORD_ENDPOINT}Academy*");
+
+        var responseBody = await response.Content.ReadAsStringAsync();
+        var results = JsonConvert.DeserializeObject<SearchResultsJson>(responseBody)!;
+
+        results.EstablishmentResults!.Establishments!.First().Urn.Should().Be("123456");
+        results.EstablishmentResults!.Establishments!.First().Name.Should().Be("Goose Academy");
+        results.EstablishmentResults.Establishments!.First().Address.Street.Should().Be("Goose Street");
+        results.EstablishmentResults.Establishments!.First().Address.Locality.Should().Be("Goose Locality");
+        results.EstablishmentResults.Establishments!.First().Address.Address3.Should().Be("Goose Address 3");
+        results.EstablishmentResults.Establishments!.First().Address.Town.Should().Be("Goose Town");
+        results.EstablishmentResults.Establishments!.First().Address.Postcode.Should().Be("GOO OSE");
+        results.EstablishmentResults!.Establishments!.First().EstablishmentType.Should().Be("Academy");
+        results.EstablishmentResults!.Establishments!.First().EstablishmentStatusName.Should().Be("Open");
+        results.EstablishmentResults!.Establishments!.First().PhaseOfEducation.Should().Be("Secondary");
```

Comment:
I would do this instead :
`        var firstResult = results.EstablishmentResults!.Establishments!.First();
        firstResult.Urn.Should().Be("123456");
        `
        _etc_
        
(A separate observation: The test might break if you add more results that match the search criteria because the first one returned might be different to what you're expecting here )

Reply:
Changes made to use variables.

Very true. With control over the data through the dummy search service it's okay at present but will keep and eye on it if we expand the test data out.